cli/create/templates/templates.go

```diff
@@ -72,6 +72,13 @@ func Instantiate(ctx *context.Ctx) error {
 		return fmt.Errorf("Failed to instantiate project template: %s", err)
 	}
 
+	spec, err := common.FindRockspec(ctx.Project.Path)
+	if err != nil {
+		return fmt.Errorf("Failed to find rockspec file in project template: %s", err)
+	}
+
+	os.Rename(spec, strings.ToLower(spec))
```

Comment:
Does it work correct only with `cartridge create` command? I suppose it would fail if we pass rockspec file by using `--spec` flag like `cartridge build MyApp --spec MyApp/MyApp-scm-2.rockspec`. Maybe we should leave some note in doc that rockspec should be passed in lower case?

Reply:
I agree, it should fall if the user passes in an incorrect (with capital letters) spec. Can you please add this note to your patch #616 ?

Reply:
Ok, done